src/test/groovy/com/infusionsoft/gradle/version/GitVersionResolverTest.groovy

```diff
@@ -44,7 +52,7 @@ class GitVersionResolverTest extends RepositoryTestCase {
     @Test
     void testGetVersionExistingReleaseWithUncommittedChanges() {
         String tagVersion = '0.0.1'
-        String expectedVersion = tagVersion + defaultTagOptions.snapshotSuffix
+        String expectedVersion = '0.0.2' + defaultTagOptions.snapshotSuffix
```

Comment:
Alright, the main use case that was discussed around this before was if someone was checking out a release version locally to debug an issue. Was just adding the snapshot to keep from potentially clobbering local release artifacts with bogus ones.